R/land_leaf.R

```diff
@@ -159,6 +159,7 @@ LandLeaf_getCalLandAllocation <- function(aLandLeaf, aPeriod) {
 #' @param aParent Parent node (needed for calibrating new techs)
 #' @author KVC September 2017
 LandLeaf_calculateShareWeights <- function(aLandLeaf, aChoiceFnAbove, aPeriod, aParent) {
+  print(aLandLeaf$mName[[1]])
```

Comment:
Was leaving this output in this function intentional?  If so, we should use `message` instead of `print`, so that calling code can suppress it if the user so desires.

However, I'd consider removing it and just adding it back in when you are doing debugging.  The cost of all that output in, say, a monte carlo run is likely to be quite large.

Hey, let me try this new code suggestion feature that GitHub has added:  

```suggestion
```